tests/autorestart/test_container_autorestart.py

```diff
@@ -142,12 +149,12 @@ def ignore_expected_loganalyzer_exception(duthosts, enum_rand_one_per_hwsku_host
     }
 
     feature = enum_dut_feature
-    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
 
     if loganalyzer:
-        loganalyzer[duthost.hostname].ignore_regex.extend(ignore_regex_dict['common'])
-        if feature in ignore_regex_dict:
-            loganalyzer[duthost.hostname].ignore_regex.extend(ignore_regex_dict[feature])
+        for duthost in duthosts:
```

Comment:
As per my understanding, this change is to ignore logs on other linecards relating to fabric ports going down. Is that correct? 

If that's the case, then can we ignore only those logs on all linecards.

The current code will ignore all logs on all linecards. Restarting container on one linecard should not result in error logs on that other linecards in the chassis (except of fabric ports going down)

Reply:
Hi @arlakshm,
Modified the code and verified. With the latest changes, 
-  Logs will be ignored on all cards, if the DUT is a supervisor node.
- Logs will be ignored only on that line card, if the DUT is a line card.

Please take a look.